src/main/scala/com/tapad/docker/DockerComposePlugin.scala

```diff
@@ -252,7 +257,7 @@ class DockerComposePluginLocal extends AutoPlugin with ComposeFile with DockerCo
       // If the compose file being used is a version that creates a new network on startup then remove that network on
       // shutdown
       if (new File(composePath).exists()) {
-        val composeYaml = readComposeFile(composePath)
+        val composeYaml = readComposeFile(composePath, variables)
```

Comment:
After that we should be able to also remove the passing of `variables` to `stopLocalDockerInstance` and `getRunningInstanceInfo`